crates/core_arch/src/arm/neon.rs

```diff
@@ -698,6 +727,229 @@ pub unsafe fn vmvnq_p8(a: poly8x16_t) -> poly8x16_t {
     simd_xor(a, b)
 }
 
+macro_rules! arm_simd_2 {
+    ($name:ident, $type:ty, $simd_fn:ident, $intrarm:ident, $intraarch:ident) => {
+        arm_simd_2!($name, $type, $type, $simd_fn, $intrarm, $intraarch);
+    };
+    ($name:ident, $type:ty, $res:ty, $simd_fn:ident, $intrarm:ident, $intraarch:ident) => {
+        #[inline]
+        #[target_feature(enable = "neon")]
+        #[cfg_attr(target_arch = "arm", target_feature(enable = "v7"))]
+        #[cfg_attr(all(test, target_arch = "arm"), assert_instr($intrarm))]
+        #[cfg_attr(all(test, target_arch = "aarch64"), assert_instr($intraarch))]
+        pub unsafe fn $name(a: $type, b: $type) -> $res {
+            $simd_fn(a, b)
+        }
+    };
+}
+
+macro_rules! arm_simd_and {
+    ($name:ident, $type:ty) => {
+        /// Vector bitwise and.
```

Comment:
Is this comment used on the functions ? One usually has to pass it to the macro that expands the function item as an argument.

Reply:
The goal was that each of the arm_simd_and expand to (example `vand_s8`):
```rust
        /// Vector bitwise and.
        #[inline]
        #[target_feature(enable = "neon")]
        #[cfg_attr(target_arch = "arm", target_feature(enable = "v7"))]
        #[cfg_attr(all(test, target_arch = "arm"), assert_instr(vand))]
        #[cfg_attr(all(test, target_arch = "aarch64"), assert_instr(and))]
        pub unsafe fn vand_s8(a: int8x8_t, b: int8x8_t) -> int8x8_t{
            simd_and(a, b)
        }
```

but I might have gotten the way macro expansion works wrong.

Reply:
All macros are gone - no longer applicable.